build_projects/shared-build-targets-utils/VersionRepoUpdater.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class VersionRepoUpdater
     {
-        private static Regex s_nugetFileRegex = new Regex("^(.*?)\\.(([0-9]+\\.)?[0-9]+\\.[0-9]+(-([A-z0-9-]+))?)\\.nupkg$");
+        private static Regex s_nugetFileRegex = new Regex("^(.*?)\\.(([0-9]+\\.)?[0-9]+\\.[0-9]+(-([A-z0-9-]+))?)(\\.symbols)?\\.nupkg$");
```

Comment:
I would suggest turning these into named groups (`(<symbols>?\\.symbols)`) instead of remember and figuring out the correct index.